sdk/ai/Azure.AI.Inference/src/Azure.AI.Inference.csproj

```diff
@@ -17,8 +17,11 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
-    <PackageReference Include="System.ClientModel" VersionOverride="1.4.0-beta.1" />
     <PackageReference Include="System.Text.Json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\core\System.ClientModel\src\System.ClientModel.csproj" />
```

Comment:
is this temporary?

Reply:
Yes, it will be updated once we release `System.ClientModel`. Otherwise, [these tests](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/ai/Azure.AI.Projects/tests/Samples/Extensions/Sample_AIInference.cs#L19) will fail.